server/controllers/clientCart.js

```diff
@@ -0,0 +1,11 @@
+const clientCartQuery = require('../database/queries/clientCartQuery')
+
+const clientCart = (req, res) => {
+  const clientId = req.params.id
+
+  clientCartQuery(clientId)
+    .then(result => res.json(result.rows))
+    .catch(err => res.status(500).json({ message: err.message }))
```

Comment:
use `next` for for server error. and never send the actual error message to the client. the message should always be "internal server error"